strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/MavenProxyRepositoryProviderTestIT.java

```diff
@@ -168,55 +141,67 @@ public void shouldBeAbleToProvideFilesFromOracleMavenRepoWithHttpsAndAuthenticat
     }
     */
 
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class })
     @Test
-    public void whenDownloadingArtifactMetadaFileShouldAlsoBeResolved(TestInfo testInfo)
+    public void whenDownloadingArtifactMetadaFileShouldAlsoBeResolved(@MavenRepository(storageId = STORAGE_ID,
+                                                                                       repositoryId = REPOSITORY_ID + "-whenDownloadingArtifactMetadaFileShouldAlsoBeResolved",
+                                                                                       setup = MavenIndexedRepositorySetup.class)
+                                                                      @Remote(url = REMOTE_URL)
+                                                                      Repository proxyRepository)
             throws Exception
     {
-        String storageId = STORAGE_ID;
-        String repositoryId = getRepositoryName(REPOSITORY_ID, testInfo);
 
-        assertStreamNotNull(storageId, repositoryId,
-                            "org/carlspring/properties-injector/1.1/properties-injector-1.1.jar");
+        String storageId = proxyRepository.getStorage().getId();
+        String repositoryId = proxyRepository.getId();
 
-        Storage storage = getConfiguration().getStorage(storageId);
-        Repository repository = storage.getRepository(repositoryId);
+        assertStreamNotNull(storageId, repositoryId,
+                "org/carlspring/properties-injector/1.1/properties-injector-1.1.jar");
```

Comment:
Cosmetic: Please, follow our coding convention and use one parameter per line. Thanks!